pkg/reconciler/managed/reconciler.go

```diff
@@ -180,7 +180,7 @@ type ConnectionDetailsFetcher interface {
 	FetchConnection(ctx context.Context, so resource.ConnectionSecretOwner) (ConnectionDetails, error)
 }
 
-// A Initializer establishes ownership of the supplied Managed resource.
+// Initializer establishes ownership of the supplied Managed resource.
```

Comment:
I started running https://writewithharper.com/docs/integrations/language-server in my editor recently and it's finding these eeeeeverywhere heh.